query_optimizer/ExecutionGenerator.cpp

```diff
@@ -830,26 +830,23 @@ void ExecutionGenerator::convertCreateIndex(
     index_attributes.emplace_back(catalog_attribute);
   }
 
-  // Corresponding to each attribute, create a copy of index description.
-  std::vector<IndexSubBlockDescription> index_descriptions;
+  // Create a copy of index description and add all the specified attributes to it.
```

Comment:
@zuyu The `input_relation_info->relation` expression returns a const reference to the relation, whereas the ExecutionGenerator needs to pass a mutable reference of CatalogRelation to CreateIndexOperator here. So, I guess this change would then not apply.